spec/requests/data_export_spec.rb

```diff
@@ -68,7 +68,28 @@
       ]
     end
 
-    it "shows all expected responses in CSV format" do
+    it "shows all expected responses in CSV format for a 1000 year range if start and end date are not provided" do
```

Comment:
It looks like the default parameters are 1 day ahead of the current date, rather than 1000 years.  Also, I can't see anything in the test that checks for the 1000 year range.

Maybe a better solution would be to export all data (with no arbitrary time limits) if no parameters are specified?

Reply:
Ah great spot, sorry when I squashed the commits I thought i'd got all references to this!
I'll amend that and the fixtures too.

Yeah had tried to just remove that, but that would have required more significant changes to the where clause that exports this. I figured it'd just be quicker to accept that 1st of Jan (before the app was built) to tomorrow, effectively always equals all time.

Reply:
Sorry was half way through mucking around with my git history and forgot i was supposed to be on a call. try now